gossip/gossip.go

```diff
@@ -656,17 +656,18 @@ func (g *Gossip) maybeWarnAboutInit(stopper *stop.Stopper) {
 		// This will never error because of infinite retries.
 		for r := retry.Start(retryOptions); r.Next(); {
 			g.mu.Lock()
+			hasConnections := g.outgoing.len() > 0 || g.incoming.len() > 0
```

Comment:
nit: kinda weird that this doesn't match the other place that does the same thing on L617. can you make the two match?